Super Team/main.css

```diff
@@ -0,0 +1,605 @@
+    0% {
+      margin-left: -180%;
+      opacity: 0;
+    }
+    100% {
+      margin-left: 0;
+      opacity: 1;
+    }
+  }
+  .lemon {
+    animation-name: bounce;
+    animation-duration: 2.5s;
+    animation-iteration-count: 1;
+    animation-timing-function: ease-in;
+  }
+  @keyframes bounce {
+    0% {
+      margin-bottom: 8vh;
+      opacity: 0.9;
+    }
+    30% {
+      margin-bottom: 0.8vh;
+      opacity: 1;
+    }
+    40% {
+      margin-bottom: 1.5vh;
+    }
+    60% {
+      margin-bottom: 0.3vh;
+    }
+    80% {
+      margin-bottom: 0.7vh;
+    }
+    100% {
+      margin-bottom: 0vh;
+    }
+  }
+  .new.black {
+    color: black !important;
+  }
+  #navbarSupportedContent {
+    text-align: right;
+  }
+  .hide-icons {
+    display: none;
+  }
+  @media screen and (min-width: 992px) {
+    .hide-icons {
+      display: inline-flex;
+    }
+  }
+  @media screen and (max-width: 992px){
+  #navbarSupportedContent {
+    background-color: rgba(0,0,0,0.5);
+  }
+  }
+/*What We Can Offer section*/
+.bg-img {
+  background-image:url(imgs/background.png);
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+.card{
+  padding: 0px 50px !important;
+}
+.style-h {
+  font-family: 'Cormorant Garamond', serif;
+}
+.padd{
+width: 70%;
+padding-top: 35px;
+}
+.center-block {
+  display: block;
+  margin-left: auto;
+  margin-right: auto;
+}
+.card-h2{
+  text-align: center;
+  color:#363535;
+  font-family: 'Arapey', serif;
+  font-size: 40px;
+  padding-bottom: 40px;
+  padding-top: 40px;
+  padding-left:30px;
+}
+.card-p{
+  text-align: center;
+  color:rgb(146, 143, 143);
+  font-size: 15px;
+  padding-left: 30px;
+}
+.column3 {
+  margin-top: 25px;
+}
+h6:before,
+h6:after {
+background-color: rgb(166, 166, 166);
+content: "";
+display: inline-block;
+height: 0.2px;
+position: relative;
+vertical-align: middle;
+width: 6%;
+margin: 10px;
+}
+/*dana's code*/
+
+.image{
+  padding: 0 !important;
+}
+.mycolumn{
+  padding: 0;
+}
+.seccolumn{
+  padding: 200px 45px 200px 140px !important;
+}
+.myrow{
+  margin: 0;
+  margin-top: 30px;
+}
+.seccolumn{
+  padding: 200px 45px 200px 140px !important;
+}
+#h1{
+  font-family: "Cormorant Garamond";
+  font-size: 40px;
+  font-weight: 200 !important;
+  opacity: 90%;
+  color: gray;
+  font-style: italic;
+}
+#lorem{
+  opacity: 95%;
+  color: gray;
+}
+.myem{
+  font-family: "Cormorant Garamond";
+  font-size: 580px;
+  font-weight: 100;
+  position:absolute;
+  font-style: italic;
+  opacity: 10%;
+  color: gray;
+  z-index: -1;
+  margin-top: -500px;
+} 
+
+.myshadow{
+  box-shadow: 0 1px 8px 4px rgba(0, 0, 0, 0.2), 0 6px 20px 0 rgba(0, 0, 0, 0.19) !important;
+}
+/*Our Team section*/
+.sec-4{
+  width: 100vw;
+}
+.lightgray-text{
+  color: rgb(166, 166, 166);
+}
+.style-h{
+  padding-top: 4%;
+  color:rgb(62, 62, 62);
+}
+.card-size{
+  margin: 3% auto !important;
+  width: 83% !important;
+  padding: 0 !important;
+}
+.card-border{
+    border-style: none !important;
```

Comment:
you are using important a lot, do you really need it this much? I don't think so

Reply:
without using it ,its doesn't work